src/app/caesar-portal/profile/profile.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component, OnInit } from '@angular/core';
+import { Response }          from '@angular/http';
+
+import { User }           from '../common/models/user';
+import { ProfileService } from './profile.service';
+
+
+@Component({
+    selector: 'caesar-profile',
+    templateUrl: './profile.component.html',
+    styleUrls: ['./profile.component.scss'],
+    providers: [ProfileService]
+})
+export class ProfileComponent implements OnInit {
+    currentUser: User;
+    error: any;
+    defaultPhoto = 'http://www.thirann.com/assets/img/newuser.jpeg';
+    visibility = true;
```

Comment:
visibility is smth abstract, let make it more readable - isVisible/isHidden or just visible/hidden will work, hidden is even better since you need to check for hidden/invisible